### DISCUSSION TITLE
Incident pulse power vs. radient exposure

### DISCUSSION THREAD

**Discussion by renesmt** [ankrh/MCmatlab#9] [Q&A] [answered]
Hello,

Firstly, I want to thank the author for this fantastic Matlab package: it runs fast, and the comment was written in such a comprehensive way!

I am just confused by the concept of 'incident pulse power', namely `model.MC.P` in the code. I was trying to reimplement the result from some paper, where they applied a 1ms pulse on a spot size ~5mm, and the 'radiant exposure' is $0.5J/cm^2$. A back-of-the-envelope calculation shows the laser power should be around 125W, `model.MC.P=125,` which would give us a ~1000K temperature rise at the spot center. I suppose it might not be safe for human skin. So, did I misunderstand anything on these concepts?

Thanks!

**Comment by ankrh** [comment, marked as answer]
Thank you for the feedback and your question!

Your calculation and use of the model.MC.P parameter seems to be correct. You might want to verify that you have correct values for mua and mus for the tissue.

If the pulse energy is high, as in your case, other processes / chemical reactions will start to take place in the tissue. These processes may be endothermic, so they draw some heat out of the system. For example, water may boil.

Is the treatment you're considering something that severely changes the tissue? Creates burns etc.?

**Reply by renesmt** [reply]
Thank you so much for your reply! :) I will double-check the settings on my end.

**Reply by leoshmu** [reply]
@ankrh thanks for the reply. We are trying to replicate numerically this study with where they evaluated bruises in skin by applying a single pulse of a 532 laser for 1 ms, using a beam diameter of 5 mm and a radiant exposure of 0.5 J/cm^2. We used the same µ_a and µs' values that they published in their work and same geometries. I think they are using a laser where one sets the 'fluence', so their input is 5J/cm^2 for 1 ms over a 5 mm beam diameter.   https://pubmed.ncbi.nlm.nih.gov/25554972/

We are just making sure that the peak incident power in MCmatlab is indeed the simple calculation of 0.5 J/cm^2 * (0.25 cm)^2*pi()/(10^-3 s) = 100 W. There may be a factor of 2 in there because the beam is guassian and not rectangulare, so the power may actually be 50W. 

Either way McMatlab is giving us a 500K temperature rise, and in their experimental and theoretical work it was on the order of 1K. So something is way off, and we are wondering if it is our conversion of radiant exposure/fluence to peak incident power.

**Reply by ankrh** [reply]
Yes, your formula for peak incident power is correct.

Check that you have the correct units for the volumetric heat capacity and the thermal conductivity. The values for usual tissues should be pretty close to that of water:
  mediaProperties(j).VHC   = 4.19; % [J cm^-3 K^-1]
  mediaProperties(j).TC    = 5.8e-3; % [W cm^-1 K^-1]

**Reply by renesmt** [reply]
We just realized that we were using a 'pencil beam' and it seems it always has a minimal radius that is not affected by model.MC.beam.NF.radialWidth.

Thank you again for your reply and patient explanation!
